cmd/pace/main.go

```diff
@@ -77,3 +80,46 @@ func CreateConfigDirAndFileIfNotExists() {
 		))
 	}
 }
+
+func CreateLastSeenCommand() {
+	lastSeenCommandFilepath := path.Join(common.ConfigPath(), common.DefaultLastSeenFilename)
+	now := time.Now()
+	if content, err := os.ReadFile(lastSeenCommandFilepath); err != nil {
+		updateLastSeen(lastSeenCommandFilepath, now)
+	} else {
+		ts, err := strconv.ParseInt(strings.Trim(string(content), "\n"), 10, 64)
+		if err != nil {
+			os.Remove(lastSeenCommandFilepath)
+			updateLastSeen(lastSeenCommandFilepath, now)
+		}
```

Comment:
When would this occur?

Reply:
if the file is somehow corrupt, if it is incorrectly changed by user for example.

Reply:
Ah, clear, thx 👍